python/cairo-core/src/cairo_core/maths.cairo

```diff
@@ -721,3 +721,37 @@ func felt252_to_bytes4_le{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
         }
     return ();
 }
+
+
+// @notice A simplified version of felt252_to_bytes4_le that always outputs 8 full bytes4 words given an input felt252.
+func felt252_to_bytes4_full{range_check_ptr}(value: felt, dst: felt*) -> felt* {
```

Comment:
is this tested?

Reply:
indirectly throught `test_felt252_array_to_bytes4_array` but there's a unit test missing 👍